client/scripts/models/NodeInfo.ts

```diff
@@ -5,12 +5,14 @@ class NodeInfo {
   public readonly chain: ChainInfo;
   public readonly url: string;
   public readonly address: string;
+  public readonly isUninitialized: boolean;
 
-  constructor(id, chain, url, address?) {
+  constructor(id, chain, url, address?, isUninitialized?) {
```

Comment:
why is this necessary for the `NodeInfo` model?

Reply:
You're right it isn't